packages/dap/src/constants.ts

```diff
@@ -1,6 +1,5 @@
 export const ROUTES = Object.freeze({
   keyConfig: "hpke_config",
-  upload: "upload",
```

Comment:
On further review, what do you think about either making this a `(TaskId, Aggregator) -> Url` inside ROUTES *OR* removing ROUTES entirely and inlining the `hpke_config` route? It feels inconsistent that one route is inlined in the code and the other is in a constant.

Reply:
I decided that inlining `"hpke_config"` made the most sense since the `ROUTES` constant was only used in one place.